gui/widget.cpp

```diff
@@ -305,14 +306,7 @@ StaticTextWidget::StaticTextWidget(GuiObject *boss, const Common::String &name,
 	_type = kStaticTextWidget;
 	_label = text;
 
-	_align = g_gui.xmlEval()->getWidgetTextHAlign(name);
-	if (g_gui.useRTL() && _useRTL) {
-		if (_align == Graphics::kTextAlignLeft) {
-			_align = Graphics::kTextAlignRight;
-		} else if (_align == Graphics::kTextAlignRight) {
-			_align = Graphics::kTextAlignLeft;
-		}
-	}
+	_align = Graphics::convertTextAlignH(g_gui.xmlEval()->getWidgetTextHAlign(name), g_gui.useRTL() && _useRTL);
```

Comment:
getWidgetTextHAlign now can return start/end, so inline it with `convertTextAlignH` and turn it into proper align.